extensions/blocks/eventbrite/style.scss

```diff
@@ -1,4 +1,13 @@
 .wp-block-jetpack-eventbrite {
+	// Make sure the widget embed container is always as tall as the
+	// default/minimum value set by Eventbrite (425px).
+	&:not( .wp-block-button ) {
+		height: auto !important;
```

Comment:
You left a good description in the PR of why this !important is needed, I don't think that's communicated in the comment here. It'd be good to be clear so that it doesn't get removed in the future when the context is lost.

Reply:
Good point! Updated the comment in 2bb979e04a91d57d6e25114dc724dcbf03edfacd 👍